src/ILCompiler.RyuJit/src/Compiler/DependencyAnalysis/RyuJitObjectWriter.cs

```diff
@@ -9,19 +9,19 @@
 using System.Runtime.InteropServices;
 
 using ILCompiler.DependencyAnalysisFramework;
-
+using Internal.JitInterface;
 using Internal.Text;
 using Internal.TypeSystem;
 using Internal.TypeSystem.TypesDebugInfo;
-using Internal.JitInterface;
+
 using ObjectData = ILCompiler.DependencyAnalysis.ObjectNode.ObjectData;
 
 namespace ILCompiler.DependencyAnalysis
 {
     /// <summary>
     /// Object writer using src/Native/ObjWriter
     /// </summary>
-    internal class ObjectWriter : IDisposable, ITypesDebugInfoWriter
+    internal class RyuJitObjectWriter : IDisposable, ITypesDebugInfoWriter
```

Comment:
Is there anything RyuJIT specific in this file?

If I were to write a new native code generator backend (say a backend completely written in managed code, for fun), this is the object writer I would use to emit the binary.